### PR TITLE
GitHub api cache

### PR DESCRIPTION
**Pull request by Shredder121**

### CODE REVIEW

File: src/test/java/com/github/shredder121/gh_event_api/handler/AbstractHandlerTest.java [hunk 1]
```diff
@@ -76,6 +77,21 @@
...
+    private static GitHub getGitHub() throws IOException {
+        GitHubBuilder builder;
+        try {
+            builder = GitHubBuilder.fromCredentials();
+        } catch (IOException noAuthorizationDetails) {
+            // This is fine, and still gives you 60 requests per hour, so one-off test runs still work
+            builder = new GitHubBuilder();
+        }
+        OkHttpClient client = new OkHttpClient();
+        client.setCache(new Cache(Paths.get(".", ".cache").toFile(), FileUtils.ONE_MB * 10));
```
Comment (johnktims) [inline src/test/java/com/github/shredder121/gh_event_api/handler/AbstractHandlerTest.java]: What about making the cache size configurable?

Comment (johnktims) [inline src/test/java/com/github/shredder121/gh_event_api/handler/AbstractHandlerTest.java]: Never mind. You're just using it in your tests.

Comment (Shredder121) [inline src/test/java/com/github/shredder121/gh_event_api/handler/AbstractHandlerTest.java]: Indeed!
Thanks for thinking along though.

Do you think that 10MB is too much? It's only for ~20 files, and they are not that big.
But github.developer.com repository, /user, I think could also be cached.

**Comment by Shredder121** [comment]
'Blocked' by kohsuke/github-api#247.

**Pull request merged**